install/helm/agones/values.yaml

```diff
@@ -92,6 +92,7 @@ agones:
       periodSeconds: 3
       failureThreshold: 3
       timeoutSeconds: 1
+    allocationBatchWaitTime: 500ms
```

Comment:
🤔 Just seeing if the idea makes sense - but should this be a single value across both the controller and allocator?

Probably not, since we duplicate other parameters between the two, but thought I would ask the questions just in case.